src/current/_includes/releases/v22.1/v22.1.21.md

```diff
@@ -0,0 +1,30 @@
+## v22.1.21
+
+Release Date: June 5, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-21-db-console-changes">DB Console changes</h3>
+
+- If a page crashes, a force refresh is no longer required to be able to see the other pages on DB Console. [#103326][#103326]
+- On the [SQL Activity > Fingerprints](../v22.1/ui-statements-page.html#statement-fingerprints-view) view, users will not see stats that have not yet been flushed to disk. [#103130][#103130]
+- Users can now request top-k stmts by % runtime on the [SQL Activity > Fingerprints](../v22.1/ui-overview.html#sql-activity) view. [#103130][#103130]
+- Added Search Criteria to the [Statements](../v22.1/ui-statements-page.html) and [Transactions](../v22.1/ui-transactions-page.html) pages. [#103130][#103130]
+
+<h3 id="v22-1-21-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in closed timestamp updates within its side-transport. Previously, during asymmetric partitions, a node that transfers a lease away, and misses a liveness heartbeat, could then erroneously update the closed timestamp during the stasis period of its liveness. This could lead to closed timestamp invariant violation, and node crashes. In extreme cases this could lead to inconsistencies in read-only queries. [#102597][#102597]
```

Comment:
suggest making 'closed timestamp' a link to https://www.cockroachlabs.com/docs/v22.1/architecture/transaction-layer#closed-timestamps

suggest making 'liveness heartbeat' a link to https://www.cockroachlabs.com/docs/v22.1/architecture/replication-layer#epoch-based-leases-table-data

Reply:
suggest changing 'asymmetric partitions' to 'assymetric network partitions' and then making 'network partitions' a link to https://www.cockroachlabs.com/docs/v22.1/cluster-setup-troubleshooting.html#network-partition